conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/config/util/package.scala

```diff
@@ -0,0 +1,21 @@
+package tech.cryptonomic.conseil.indexer.config
+
+import scala.util.Try
+
+package object util {
```

Comment:
I think you can move contents ot this package object to the ConfigUtil object

Reply:
Done, named it `ConfigDepthUtil` since it solely relates to `Depth`.

Reply:
I meant that you can move it to the different file which is named `ConfigUtil`, it has other objects inside and you can name it just `Depth`

Reply:
done